### DISCUSSION TITLE
List of containers and option to remove

### DISCUSSION THREAD

**Discussion by devgrpereira** [rancher-sandbox/rancher-desktop#2484] [General]
As Rancher Desktop also has dockerd and containerd, we used it to replace Docker Desktop with Visual Studio 2019 and 2022. We didn't find a visual option to list the containers and option to exclude them. 
Do you know where you have or if you have the possibility to have this option?

**Comment by tanguyantoine** [comment]
As of today, Rancher Desktop does not have this functionnality yet.
